src/relay/charpane.ash

```diff
@@ -4532,112 +4532,112 @@ buffer addGroup(string[int] rowHTMLs, string className) {
 buffer addRow(string[int] bricks, string[int] styleInfos) {
 //adds bricks or groups of bricks into horizontal rows
 	buffer buff;
-	string brick;
 	int totalStars = starCount(styleInfos) + count(bricks) - count(styleInfos);
-	//append(buff, '<div class="brick_row">');
 	foreach i,s in bricks {
-		brick = s;
-		if(chitBricks contains s) {
-			brick = chitBricks[s];
-		} else {
-			if(length(s) == 0 || substring(s,0,1) != "<")
-				brick = "";
-		}
-		if(length(brick) > 0) {
-			append(buff, '<div class="brick_holder" '+stylize(styleInfos[i],totalStars)+'>'+brick+'</div>');
-		}
+		append(buff, '<div class="brick_holder" '+stylize(styleInfos[i],totalStars)+'>'+s+'</div>');
```

Comment:
Minor thing I didn't notice before: in most cases ChIT will do `buff.append(whatever)` instead of `append(buff, whatever)`. I'm not particularly bothered by things being done differently in some places though.

Reply:
I'm not sure if you're actually asking for this to be changed, but since you made the comment I'll take a look. Consistent style choices are nice.

Reply:
I actually tried to change it from buff.append(whatever), and mafia started giving me errors. Not sure what's going on, but it doesn't look like this change works.

Reply:
Weird. Well, it's not a big issue, so that's fine.